src/frontend/src/catalog/catalog_service.rs

```diff
@@ -95,21 +95,17 @@ pub trait CatalogWriter: Send + Sync {
         job_type: TableJobType,
     ) -> Result<()>;
 
-    async fn alter_source_column(&self, source: PbSource) -> Result<()>;
-
     async fn create_index(
         &self,
         index: PbIndex,
         table: PbTable,
         graph: StreamFragmentGraph,
     ) -> Result<()>;
 
-    async fn create_source(&self, source: PbSource) -> Result<()>;
-
-    async fn create_source_with_graph(
+    async fn create_source(
```

Comment:
Unify multiple `create_source` into one function, since it corresponds to one meta rpc. 

Similar for `alter_source`